ra_aid/agent_utils.py

```
@@ -353,42 +529,48 @@ def run_planning_agent(
-        key_snippets=get_memory_value('key_snippets'),
-        research_only_note='' if config.get('research_only') else ' Only request implementation if the user explicitly asked for changes to be made.'
+        key_facts=get_memory_value("key_facts"),
+        key_snippets=get_memory_value("key_snippets"),
+        work_log=get_memory_value("work_log"),
+        research_only_note=""
+        if config.get("research_only")
+        else " Only request implementation if the user explicitly asked for changes to be made.",
     )
 
-    # Set up configuration
+    config = _global_memory.get("config", {}) if not config else config
```

Ruff (0.8.2)

Use `config if config else _global_memory.get("config", {})` instead of `_global_memory.get("config", {}) if not config else config`

Replace with `config if config else _global_memory.get("config", {})`

(SIM212)